stacks/SiteStack.ts

```diff
@@ -146,6 +146,7 @@ export function SiteStack({ stack }: StackContext) {
                     "cognito-idp:AdminDeleteUser",
                     "cognito-idp:AdminCreateUser",
                     "cognito-idp:ListUsersInGroup",
+                    "cognito-idp:AdminListGroupsForUser",
```

Comment:
Think we need to add "cognito-idp:AdminRemoveUserFromGroup" as well as I forgot to add that in.